opentrons-ai-client/src/organisms/ProtocolFormatSection/index.tsx

```diff
@@ -49,9 +55,16 @@ export function ProtocolFormatSection(): JSX.Element | null {
         defaultValue={PYTHON}
         rules={{ required: true }}
       />
-      <Banner type="informing" marginBottom={SPACING.spacing16} height={SIZE_3}>
-        {t('pd_prompt_warning')}
-      </Banner>
+      {/* Only show banner when Protocol Designer is selected */}
+      {selectedFormat === PD && (
+        <Banner
+          type="informing"
+          marginBottom={SPACING.spacing16}
+          height={SIZE_3}
```

Comment:
we stopped using `SIZE_number` const

```suggestion
          height="4rem"
```

Reply:
Thanks!